services-launcher/src/main/resources/application.yaml

```diff
@@ -25,25 +25,58 @@ spring:
 # These are built from launcher.defaults with SERVICE substituted for the service.name
 # and any other overrides specified will replace the default
 launcher:
-  default-service:
-    root: palisade-services
-    name: default
-    target: SERVICE/target/SERVICE-0.4.0-SNAPSHOT-exec.jar
-    config: SERVICE/classes
-    profiles: default
-    log: SERVICE.log
+  # search up path for a named root directory
+  root: palisade-services
+  # library root relative to launcher.root above
+  library-root: ../palisade-examples/example-library
+  # java -cp [..]policy-service.jar:[...]example-library.jar -Dspring.config.location=policy-service/target/classes/ -Dspring.profiles.active=jvm, -Dloader.main=[...].PolicyApplication [...].JarLauncher
   services:
-    -
-      name: audit-service
-    -
-      name: data-service
-    -
-      name: discovery-service
-    -
-      name: palisade-service
-    -
-      name: policy-service
-    -
-      name: resource-service
-    -
-      name: user-service
+    audit-service:
+      classpath: audit-service/target/audit-service-0.4.0-SNAPSHOT-exec.jar:${launcher.library-root}/target/example-library-0.4.0-SNAPSHOT-jar-with-dependencies.jar
```

Comment:
Is it possible to avoid having the version string hard coded in all these lines? Could they be read from a Maven property?

Reply:
This has now been adjusted so the service version uses the maven property. The example-library jar will not be using this maven property value.